src/harness/simple_crl_server.py

```diff
@@ -26,11 +26,6 @@
 
 DEFAULT_CRL_URL = 'http://localhost:9007/ca.crl'
 DEFAULT_CRL_SERVER_PORT = 80
-CERT_TYPE = {
```

Comment:
What's the motivation for these changes?

Reply:
To avoid   human error, CRL Server is now enhanced    to identify the cerificate type automatically based on issuer type to   determine whether it is intermediate CA or leaf ceritficate during   certificate revocation.